front/tests/assets/paced-train/paced_trains.json

```
@@ -142,5 +142,42 @@
         "speed_limit_tag": { "value": "HLP" }
       }
     ]
+  },
+  {
+    "train_name": "paced train with exceptions",
+    "labels": ["Paced-Train-Tag-1", "SS-NS", "Valid"],
+    "rolling_stock_name": "SLOW_RS_E2E",
+    "start_time": "2025-06-26T00:00:00Z",
+    "path": [
+      { "id": "id227", "deleted": false, "uic": 6, "secondary_code": "BV" },
+      { "id": "id228", "deleted": false, "uic": 5, "secondary_code": "BV" }
+    ],
+    "schedule": [
+      {
+        "at": "id228",
+        "arrival": null,
+        "stop_for": "P0D",
+        "reception_signal": "OPEN",
+        "locked": false
+      }
+    ],
+    "margins": { "boundaries": [], "values": ["0%"] },
+    "initial_speed": 0,
+    "comfort": "STANDARD",
+    "constraint_distribution": "MARECO",
+    "speed_limit_tag": null,
+    "power_restrictions": [],
+    "options": { "use_electrical_profiles": true },
+    "paced": { "time_window": "PT2H", "interval": "PT1H" },
+    "category": "FREIGHT_TRAIN",
+    "exceptions": [
+      {
+        "key": "e0f2d454-ccaa-4df6-8a57-39c28c904976",
+        "disabled": false,
+        "start_time": {
+          "value": "2025-06-26T15:00:00Z"
```

nit: so the added exception is inserted among the current occurrences
```suggestion
          "value": "2024-10-15T04:23:00Z"
```

---

04:23 is not between the existing occurrences. (00:00 and 01:00)

I put 2025-06-26T00:37:00Z, tell me if this is what you meant :)

---

Yes perfect :)